data/mods/Generic_Guns/gg_ammo_override.json

```diff
@@ -426,4 +426,4 @@
         "count" : 10,
         "effects" : ["INCENDIARY"]
     }
-]
```

Comment:
vim (:+1:) adds the newline, it isn't in the current master branch.

A newline at the end of a text file is standard in the Unix world. See for example http://unix.stackexchange.com/questions/18743.

Basically: Unix considers a line to be *terminated* by a newline, which is thereby part of the line. Windows considers the newline to be a line *separator*, the last line does not need to be separated from anything and therefor does not need a newline here.

Editors use/enforce/encourage whatever they like, many can be configured for one or the other behavior.

> e: cross-checking with the original file, I don't see a trailing newline (in the original, I mean - both my PR and the master are 429 lines).

Yes, some Unix tools may consider the last line to be technically "incomplete" without the newline character. The program you used for line counting does apparently not care about this. But other line counting tools may simply count the number of newline characters and therefor give a different result.